app/grapefruit/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+edition = "2021"
+readme = "README.md"
+name = "grapefruit"
```

Comment:
_suuuuuuper_ nitpicky, feel free to ignore me: it's a little bit weird to me that the `name` key is after other metadata...at first glance it looked like the crate didn't _have_ a name.